frontend/src/screens/AddCourse/services/course.service.ts

```diff
@@ -1,10 +1,27 @@
 import { callApi } from 'helpers/api.helper';
+import lectures from '../models/lectures.json';
+import { ICourse } from '../models/ICourse';
 
 export const getLecturesByAuthor = async (id: string) => {
   const response = await callApi({
-    endpoint: `/api/lectures/author/${id}`,
+    endpoint: `api/course/lectures/user/${id}`,
     type: 'GET'
   });
+  const result = response.json();
+  return result;
+};
+
+/* export const getLecturesByAuthor = async (id: string) => new Promise(resolve => {
+  setTimeout(() => resolve({
+    lectures
+  }), 500);
+}); */
```

Comment:
Do you need this comment?

Reply:
yes, this method exists to fetch data from mock